### ISSUE TITLE
[Tarea 3] SVC Muy lento

### ISSUE THREAD

**Issue by Brocoleta** [UC-IIC2613/Syllabus#166]
Hola estoy tratando de hacer la pregunta 3 de la tarea, pero tengo el problema que el svc.fit corre demasiado lento, nunca termina, trate de reducir el training set a la mitad pero tampoco funciona. Nose si estoy haciendo algo mal o realmente es muy lento

**Comment by chris-carstens** [comment]
Hola! @Brocoleta 
Primero que todo, estás usando la GPU? Te recomiendo usarla para la tarea
Según las combinaciones de modelos que hagas, efectivamente hay algunos que te pueden demorar mucho, según los parámetros que uses, es normal.

**Comment by chris-carstens** [comment]
En particular con SVM,  puede ser que tengas un valor de slack muy grande y el SVM se demore demasiado pues tiene mucha holgura para mover los planos. Parte por valores pequeños de slack y anda incrementando.
Además, verifica cunánto se te demora en correr un único fit.

**Comment by Brocoleta** [comment]
Hola, si estoy ocupando un GPU, acelere bastante el proceso escalando el set con las librerias de sklearn.preprocessing, pero veo que esto afecta el accuracy, nose si esta bien?

**Comment by chris-carstens** [comment]
Sí estaría bien, siempre y cuando no te afecte contundentemente el desempeño de tus modelos. Además, si justificas correctamente en esa pregunta el correcto uso de la librería y por qué vale la pena perder accuracy, mejor aún.
